### DISCUSSION TITLE
More than one custom colour configurations

### DISCUSSION THREAD

**Discussion by hasantahir** [ahrm/sioyek#260] [General]
I am really enjoying the custom dark mode configuration. I would also dearly appreciate a custom colour mode for the bright mode where I can change the background colour from white. At the moment, the flag `background_color    0.92 0.89 0.83` only changes the background colour of the app and not the document:

Bright Mode:
```
background_color    0.92 0.89 0.83
```
<img width="1073" alt="image" src="https://user-images.githubusercontent.com/11236148/170894869-5a36a01e-cfc2-4951-952b-12c1652b1742.png">

Custom Dark Mode:
```
dark_mode_contrast 1
custom_background_color 0.180 0.204 0.251
custom_text_color 0.847 0.871 0.914
```
<img width="1036" alt="image" src="https://user-images.githubusercontent.com/11236148/170894878-5dc7d478-bc59-4c96-9d37-955926fa8370.png">

**Comment by ahrm** [comment]
I think this is best handled with an external script as it is a relatively niche feature. See https://github.com/ahrm/sioyek/issues/206.

**Reply by hasantahir** [reply]
Thanks for the suggestion, by changing the `prefs_user.config` file to:
```
custom_text_color 0 0 0 
custom_background_color    0.92 0.89 0.83
```
I am able to get the desired display:


<img width="1654" alt="image" src="https://user-images.githubusercontent.com/11236148/170945297-0ada4fac-6e25-4733-8ee5-2dc150a29532.png">

How will an external script be able to handle this?

**Reply by ahrm** [reply]
Yes.

**Reply by hasantahir** [reply]
Thanks. I will give it a shot and report any progress here.
